sync-tool/bin/update

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env php
+<?php
+
+require_once __DIR__ . '/../vendor/autoload.php';
+require_once __DIR__ . '/../UpdateFromPerlSrc.php';
+
+
+\Yaoi\Cli\Command\Runner::create(new UpdateFromPerlSrc)->run();
```

Comment:
I suggest putting the ``UpdateFromPerlSrc`` in a namespace as it is one of our classes